website/src/pages/showcase/index.js

```diff
@@ -35,9 +61,13 @@ function Showcase() {
               Add your site!
             </a>
           </p>
+          {TAGS.map((tag) => (
+            <ToggleTags tag={tag} key={tag} change={() => clickTag(tag)} />
```

Comment:
we want the tag checkbox to be "controlled", this means that you will provide an explicit state to it

```jsx
<input
            type="checkbox"
            checked={seletedTags.include(currentTag)}
            onChange={() => toggleTag(currentTag)} 
/>
```